### DISCUSSION TITLE
纯小白求助

### DISCUSSION THREAD

**Discussion by t1yifang** [QingdaoU/OnlineJudge#464] [Q&A]
请问大家是怎么进行二次开发的呢 我在ubuntu上部署了FE和onlinejudge,在ubuntu上只能看到前端页面，也进不去后台管理，课设需要在青岛oj上开发，不太了解这种架构的开发方式，部署完了，怎么对前端和后端二次开发，有没有对这个项目有过二开经验的大佬，求有偿指导

**Comment by mikucat0309** [comment]
青岛 OJ 的架构在这里 https://opensource.qduoj.com/#/en/development/architecture
二次开发总得有个目标吧
